workflows/reference/inputs/hic-reference-inputs.json

```diff
@@ -0,0 +1,8 @@
+{
+    "make_hic_reference.reference_fa_url": "https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.15_GRCh38/seqs_for_alignment_pipelines.ucsc_ids/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz",
+    "make_hic_reference.reference_fa_name": "GRCh38_no_alt.fa.gz",
+    "make_hic_reference.exclude_list_url": "https://github.com/Boyle-Lab/Blacklist/raw/master/lists/hg38-blacklist.v2.bed.gz",
+    "make_hic_reference.exclude_list_name": "hg38-blacklist.v2.bed.gz",
+    "make_hic_reference.restriction_sites":  ["^GATC", "^GATC", "A^GATCT", "A^AGCTT", "G^ANTC", "AT^CGAT"],
+    "make_hic_reference.restriction_sites_names":  ["mboi", "dpnii", "bglii", "hindiii", "hinfi", "clai"]
```

Comment:
Should these be moved to the defaults of the workflow? In what situations would a user want to use something different?

Reply:
Restriction sites are custom to the experiment. I put several of the most common ones here, but there's no reason someone couldn't be using a different cut site.